README.md

```diff
@@ -155,10 +293,31 @@ using LoggingExtras
 using Logging
 using HTTP
 
-function not_HTTP_message_filter(level, message, _module, group, id, file, line; kwargs...)
-	_module != HTTP
+function not_HTTP_message_filter(log)
+	log._module != HTTP
+end
+
+global_logger(EarlyFilteredLogger(not_HTTP_message_filter, global_logger()))
```

Comment:
I've been thinking we should have a nice function to do this wrapping operation. This is a particular reason I'd like to be able to instantiate filters and transformers without talking about the sink.